prime-dotnet-webapi/ApiDbContext.cs

```diff
@@ -42,11 +42,11 @@ public ApiDbContext(DbContextOptions<ApiDbContext> options, IHttpContextAccessor
         public DbSet<AccessAgreementNote> AccessAgreementNotes { get; set; }
         public DbSet<EnrolmentCertificateNote> EnrolmentCertificateNotes { get; set; }
 
-        public DbSet<TermsOfAccess> TermsOfAccess { get; set; }
+        public DbSet<AccessTerm> AccessTerm { get; set; }
```

Comment:
Should this be pluralized to AccessTerms?

Reply:
yes, good catch